.github/workflows/pr_build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           make V=1 pkgs
           COMMIT_ID=$(git describe --abbrev=8 --always)
           echo "VERSION=0.0.0-pr$PR_ID-$COMMIT_ID" >> $GITHUB_ENV
-          echo "TAG=evebuild/danger:pr$PR_ID" >> $GITHUB_ENV
+          echo "TAG=evebuild/publish_pr_build:pr$PR_ID" >> $GITHUB_ENV
```

Comment:
Why are we changing this?

Reply:
To make more consistent file name with workflow name

Reply:
This is a bad idea -- there has to be some indication that using this container could literally harm your computer. The reason it is called danger is not to be cute.

Reply:
@rvs ok, I will put some comments about that.